package.json

```diff
@@ -1,17 +1,22 @@
 {
   "version": "1.0.0",
   "scripts": {
```

Comment:
You are missing commit script, see https://github.com/seznam/ima/blob/master/package.json#L59